package.json

```diff
@@ -29,20 +31,25 @@
     "renders"
   ],
   "author": "Brian Vaughn <brian.david.vaughn@gmail.com>",
-  "contributors": ["Brian Vaughn <brian.david.vaughn@gmail.com>"],
+  "contributors": [
+    "Brian Vaughn <brian.david.vaughn@gmail.com>"
+  ],
   "license": "MIT",
-  "dependencies": {
-    "jest": "^21",
-    "react": "^16.5"
-  },
+  "dependencies": {},
   "devDependencies": {
     "babel-jest": "^23.6.0",
     "babel-preset-env": "^1.7.0",
     "babel-preset-react": "^6.24.1",
     "eslint": "^5.10.0",
     "eslint-config-prettier": "^3.3.0",
     "eslint-plugin-prettier": "^3.0.0",
+    "jest": "^21",
     "prettier": "^1.15.3",
+    "react": "^16.5",
     "react-dom": "^16.5"
+  },
+  "peerDependencies": {
+    "jest": ">21",
+    "react": ">16.5"
```

Comment:
Why the change to `>` instead of `^`? I think `^` seems more appropriate for now.